client/src/StarIcon.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const StarIcon = props => {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      fill={props.fill ? "gold" : "grey"}
+      className="feather feather-star"
```

Comment:
I don't think you need the feather classes here (unless you've brought in feather's stylesheet and are thus referencing them).